core/CMakeLists.txt

```diff
@@ -6,10 +6,23 @@ IF (NOT Kokkos_INSTALL_TESTING)
   ADD_SUBDIRECTORY(src)
 ENDIF()
 
+FUNCTION(KOKKOS_ADD_BENCHMARK_DIRECTORY DIR_NAME)
```

Comment:
Why did you decide to add it here rather than next to the definition of `KOKKOS_ADD_TEST_DIRECTORIES`?
(Not asking for any change at this time)

Reply:
It was in the spirit of staying away from Trilinos / Tribits in our CMake (`cmake/kokkos_tribits.cmake` precisely).
I am thinking about creating something like `cmake/kokkos_benchmark.cmake` though (with the content of this function + some general bits from `cmake/core/perf_test/CMakeLists.txt`).

Reply:
Also keeping it local since it's the only call site for now, although I agree that it looks out of place here.